bolt/src/main/java/com/slack/api/bolt/AppConfig.java

```diff
@@ -84,11 +84,36 @@ public boolean isDistributedApp() {
         return clientId != null && clientSecret != null;
     }
 
+    // --------------------------
+
+    @Builder.Default
+    private boolean oAuthInstallPathEnabled = false;
+
+    @Deprecated // will be removed in v2.0 - use oAuthInstallPathEnabled
     @Builder.Default
     private boolean oAuthStartEnabled = false;
+
+    @Deprecated
+    public boolean isOAuthStartEnabled() {
```

Comment:
Explicitly defining getter/setter methods enables us to overwrite the auto-generated methods by Lombok.